README.md

```diff
@@ -351,6 +351,36 @@ Then run the tests:
 
     $ npm run test
 
+## Contributors
```

Comment:
Maybe we should add a new level-1 header "Contributing" and also link to or [How to help](https://github.com/almende/vis/blob/master/misc/how_to_help.md) page?